azure-java-migration-copilot-client/src/main/java/com/azure/migration/java/copilot/service/resource/ApplicationConfiguration.java

```diff
@@ -0,0 +1,73 @@
+package com.azure.migration.java.copilot.service.resource;
+
+import com.fasterxml.jackson.annotation.JsonClassDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.victools.jsonschema.generator.*;
+import com.github.victools.jsonschema.module.jackson.JacksonModule;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+@JsonClassDescription("The Application configuration")
+public class ApplicationConfiguration {
```

Comment:
cf manifest file is starts with 
```yaml
---
applications:
- name: xxxx
```
it's an array object, how should we deal with multiple apps scenario?

Reply:
Yes I thought this issue, there should be lots of work need to do, we support single one first